week_1/Array/product_of_array_except_self/product_of_array_except_self.js

```diff
@@ -0,0 +1,33 @@
+// Given an integer array nums, return an array answer such that answer[i] is equal to the product of all the elements of nums except nums[i].
+
+// The algorithm must run in O(n) time and can't use the division operator.
+/**
+ * @param {number[]} an array of numbers
+ * @return {number[]} an array of products
+ */
+
+const productExceptSelf = (nums) => {
+	let fromLeft = [];
```

Comment:
```suggestion
	const fromLeft = [];
```

since we're mutating just the indexes and not the entire variable, this should be `const`.